### DISCUSSION TITLE
Google Fonts

### DISCUSSION THREAD

**Discussion by pitiflo** [inkstitch/inkstitch#1487] [Q&A]
I'm thinking about digitizing some google fonts when I need one for a project do you think there is an issue with licence ?

**Comment by Augusa** [comment]
No problem with the google font license. They are all under OFL and several fonts in Inkscape are on this license. You just nead to change a little the name. 
We are a group of french embroiderers and we already create fonts for Inkstitch. I suggest taking a look in the trial versions of the future Inkstitch version to see which fonts will be there to avoid creating an existing font.
It is also possible to say here your project and we tell you if it is "free".
We had to solve a lot of questions for creating fonts for Inkstitch. Of course it is possible to share our experiences.

**Comment by pitiflo** [comment]
I'm French too so that can probably help. I did installed the last beta and the last font pushed me to ask if it was possible and if some people where interested

**Comment by Augusa** [comment]
Interested persons can read this tutorial in english here
https://inkstitch.org/tutorials/font-creation/
It is a translation and adaptation of my french tutorial
https://lyogau.over-blog.com/2020/12/inkstitch-creer-une-police-de-caracteres-brodes.html
For more complete information contact me through my blog.
